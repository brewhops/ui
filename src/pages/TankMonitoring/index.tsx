import React from 'react';

export const TankMonitoring: React.FC = () => {
  return <div>Tank Monitoring</div>
}
  