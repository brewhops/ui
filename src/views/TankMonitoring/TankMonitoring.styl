@import '../../styles/breakpoints'

#content {
  display: grid;

  +greater-than(desktop) {
    grid-template-columns: 1fr;
  }

  grid-template-columns: 1fr;
  grid-template-areas: 'info';

  +less-than(tablet) {
    grid-template-columns: 98vw;
    grid-template-areas: 'info';
  }

  justify-items: center;
}