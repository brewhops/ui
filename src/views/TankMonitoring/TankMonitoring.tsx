import React from 'react';
import './TankMonitoring.styl';

export const TankMonitoring: React.FC = () => {
  return <div>Tank Monitoring</div>
}
  