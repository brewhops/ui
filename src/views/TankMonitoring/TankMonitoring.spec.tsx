import React from 'react';
import TankMonitoring from './TankMonitoring';
import { shallow } from 'enzyme';

describe('src/pages/TankMonitoring/TankMonitoring.tsx', () => {
  it('renders the Tank Monitoring page', () => {
    expect(shallow(<TankMonitoring/>)).toMatchSnapshot();
  })
});