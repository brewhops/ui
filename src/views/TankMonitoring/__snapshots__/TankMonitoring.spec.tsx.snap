// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/TankMonitoring/TankMonitoring.tsx renders the Tank Monitoring page 1`] = `ShallowWrapper {}`;
